lms/templates/static_templates/ferie_bez_nudy.html

```diff
@@ -0,0 +1,549 @@
+            <div class="webinar__block__content my-3 d-flex justify-content-between flex-wrap" data-start="January 12, 2021 12:00:00">
+                <!-- add at the end class "webinar__block__content-video" new class "inactive" for deactivate webinar to available soon -->
+                <button class="webinar__block__content-video col" type="button" aria-pressed="false" aria-label="Otwórz webinar Podstawy programowania Scratch" data-toggle="modal" data-target="#modal-webinar-video-1" style="background-image: url('/static/images/webinars/podstawy-programowania-scratch.jpg');">
+                    <span class="available-soon">Dostępny wkrótce</span>
+                </button>
+                <div class="webinar__block__content-description col p-3">
+                    <span class="webinar__block__content-description--date font-weight-bold">Webinar</span>
+                    <h3 class="webinar__block__content-description--title my-1">Podstawy programowania Scratch</h3>
+                    <p class="webinar__block__content-description--text">
+                        Podstawy programowania to wirtualny warsztat dla dzieci zainteresowanych technologią. Zajęcia prowadzi Maciej Kowalski – znawca programowania i fan dobrej jakości kodu. Wspólnie z uczniami szkół podstawowych pobawi się językiem Scratch i krok po kroku stworzy różne obiekty w świecie kultowego Minecrafta. <br>Gwarantujemy dużą dawkę wiedzy i dobrego humoru, a na zakończenie radość i satysfakcję. Gotowi na zabawę w programistę?
+                    </p>
+                    <!-- add at the end class "webinar__block__content-more" new class "inactive" for deactivate webinar to available soon -->
+                    <p class="webinar__block__content-more">
+                        <button class="webinar__block-link ff-OpenSans font-weight-bold js-toogle-more" data-toggle="collapse" data-target="#collapse-webinar-1" type="button" aria-expanded="false" aria-controls="collapse-webinar-1" aria-describedby="collapse-webinar-1">Więcej informacji <span class="sr-only">o Podstawy programowania Scratch</span></button><i class="fas fa-chevron-down ml-1" aria-hidden="true"></i>
+                    </p>
+                </div>
+                <div class="webinar__block__content--more-info p-3 col-12 collapse" id="collapse-webinar-1">
+                    <div class="d-flex flex-row flex-wrap mb-1">
+                        <h4 class="webinar__block__content--more-info-h4 mr-2">Prowadzący: </h4><span><strong>mgr inż. Maciej Kowalski</strong>, Ośrodek Przetwarzania Informacji – Państwowy Instytut Badawczy</span>
+                    </div>
+                    <div class="invisible">
+                        <h4 class="webinar__block__content--more-info-h4">Opis:</h4>
+                        <p class="webinar__block__content--more-info-descr">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce non erat at quam rutrum pulvinar sit amet quis tellus.
```

Comment:
Lorem ipsum :)

Reply:
to są elementy ukryte, miały być dodatkowe opisy, materiały od autorów, ale mało kto co dostarczył, zostało na ewentualne zaktualizowanie.